### PR TITLE
gRPC Tutorial: Add client code gen details

### LINKED ISSUE #24819: Feedback for gprc client documentation
I found it incredibly hard to generate the client and reference it successfully. The interaction of class names generated by gprc with C# namespaces makes it difficult. The only way I got my code to work was by following the instructions here: https://grpc.io/docs/languages/csharp/quickstart/. The key point in that article was pointing me to the location of the generated files. There I could see the exact code that gprc had generated and then it was breeze. I suggest updating the documentation to add that.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1d298a6c-8ddc-dbc8-d219-0fa7138442c2
* Version Independent ID: b96be346-958a-cfcd-d80a-c352d64223e8
* Content: [Create a .NET Core gRPC client and server in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/tutorials/grpc/grpc-start?view=aspnetcore-6.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/grpc/grpc-start.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/grpc/grpc-start.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @JamesNK
* Microsoft Alias: **jamesnk**

### PR DESCRIPTION
**Pull request by wadepickett**
[Internal Review](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/grpc/grpc-start?view=aspnetcore-6.0&branch=pr-en-us-25641&tabs=visual-studio)

Fixes #24819 

Added a little more detail on what grpc.tools is generating so it is clear where to find the two .cs files it creates based on the .proto file per customer request.

### CODE REVIEW

**Comment by wadepickett** [comment]
@James, I didn't want to go into too much detail on the generated files and then bog down the tutorial, but I agree with the customer that some more was needed.  See if that feels like the right balance to you.

**Comment by JamesNK** [comment]
This is what the doc looks like now:

image

It's a bit confusing about what are required steps to get the app working vs additional information. What do you think of putting the new content in an [!INFO] to make it clearer?

**Comment by wadepickett** [comment]
Good idea.  You are right, it was getting too busy attempting to put in inline.

**Comment by wadepickett** [comment]
@JamesNK.  I put the new content in a [!Note] section.   Look OK?

**Comment by wadepickett** [comment]
Hi @JamesNK, is this OK to publish?  ( I didn't see a review approval yet. )

I put the new content in a [!Note] section as was suggested.

[Here is a review link directly to that section](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/grpc/grpc-start?view=aspnetcore-6.0&branch=pr-en-us-25641&tabs=visual-studio#create-the-greeter-client)

Thanks so much for the feedback and help.

File: aspnetcore/tutorials/grpc/grpc-start.md [hunk 1]
```diff
@@ -523,11 +531,18 @@ dotnet add GrpcGreeterClient.csproj package Grpc.Tools
 
 ### Create the Greeter client
 
-Build the client project to create the types in the `GrpcGreeter` namespace. The `GrpcGreeter` types are generated automatically by the build process.
+* Build the client project to create the types in the `GrpcGreeterClient` namespace.
 
-Update the gRPC client `Program.cs` file with the following code:
+The `GrpcGreeterClient` types are generated automatically by the build process. The tooling package [Grpc.Tools](https://www.nuget.org/packages/Grpc.Tools/) generates the following files based on the *greet.proto* file:
```
Comment (JamesNK) [inline aspnetcore/tutorials/grpc/grpc-start.md:536]: Missing note here

**Review by JamesNK — CHANGES_REQUESTED** [review]

File: aspnetcore/tutorials/grpc/grpc-start.md (hunk 1 above)
Comment (wadepickett) [reply aspnetcore/tutorials/grpc/grpc-start.md]: Good catch!  Thanks, I should have added it to previous .net versions in the doc as well, which is in now and corrected.

**Review by JamesNK — APPROVED** [review]

**Pull request merged**